test/controllers/agent/AgentContactDetailsUpdatedControllerSpec.scala

```diff
@@ -40,7 +40,7 @@ class AgentContactDetailsUpdatedControllerSpec extends SpecBase {
         val view = application.injector.instanceOf[AgentContactDetailsUpdatedView]
 
         status(result) mustEqual OK
```

Comment:
Aashu your ticket is to add dynamic content so this test should show specific information when a clients details have been selected. This test is for "when client details exist" so the clientSelected value should be set to true and there needs to be another test added for when clientSelected is false.